imagedephi/models/rules.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
```

Comment:
You can feel free to put this into whatever file you'd like. Maybe it should go back into `rules/`?

Reply:
The `rules` module was removed. Is there an issue with defining the enum here?